react/features/filmstrip/components/web/Filmstrip.js

```diff
@@ -243,6 +324,19 @@ class Filmstrip extends PureComponent <Props> {
     constructor(props: Props) {
         super(props);
 
+        this._filmstripRef = React.createRef();
+        this._dragRef = React.createRef();
+
+        this._dragRef.current = {
+            down: false,
```

Comment:
Why don't we store those things in the state?

Reply:
I initially used it differently, but yes, they can be stored in state now